vulnerable_people_form/form_pages/dietary_requirements.py

```diff
@@ -8,7 +8,7 @@
     form_answers,
     get_errors_from_session,
     request_form,
-    update_session_answers_from_form,
+    update_session_answers_from_form_for_enum,
```

Comment:
trailing comma should be removed - probably should go through all the files in the commit to double check this as i have seen it in quiet a few places

Reply:
The trailing comma is part of general good practice standards.

Reply:
I have tended to see it used in collections or lists but no so much else where either thats fine